app/assets/stylesheets/pictures.scss

```diff
@@ -0,0 +1,37 @@
+// Place all the styles related to the pictures controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+article{
+  background: white;
+  padding: 0;
```

Comment:
Properties should be ordered background, border, border-radius, display, margin, padding